test/ordered_set_test.dart

```diff
@@ -305,5 +305,41 @@ void main() {
         expect(orderedSet.toList().join(), 'cdab');
       });
     });
+
+    group('reversed', () {
```

Comment:
removeAll test?

Reply:
Forgot that I added that method too, now the test is added.